warehouse/query-core/src/main/java/datawave/query/jexl/JexlNodeFactory.java

```diff
@@ -268,7 +269,7 @@ public static JexlNode createNodeFromValuesForField(String field, IndexLookupMap
     }
 
     /**
-     * This method is currently only used from the QueryModelVisitor to expand a binary node into multiple pairs. If one side contains the lenient marker, then
+     * This method is currently only used from the QueryModelVisitor to expand a binary node into multiple pairs. If one side contains the strict marker, then
```

Comment:
There is not a better answer than 'because it's always been here'.  I have no problems moving it.

Reply:
well, actually this method seems consistent with a bunch of the other methods in this class.  Unless you have a good reason, I am tending to lean towards leaving it in here.

Reply:
Assuming I'm looking at the same methods as you, the difference is that those methods are used in multiple places, not just the QueryModelVisitor.  I am still in favor of moving this method since it is specific to QueryModelVisitor.

Reply:
Created a PR with a potential refactor: https://github.com/NationalSecurityAgency/datawave/pull/2022

Reply:
But it could be used elsewhere....

Reply:
well.... since you went through all that trouble.  Thanks.

Reply:
Where else would we use that method though?  If you genuinely think this method is useful outside of the QueryModelVisitor, then I will close my PR (which was no trouble at all btw).

Reply:
I merged in your PR.  I verified it was not being used elsewhere